data/themes/default/style.css

```diff
@@ -540,28 +540,47 @@ TrackContainerView QLabel
 /* instrument pattern */
 PatternView {
 	color: rgb( 119, 199, 216 );
+	qproperty-mutedColor: rgb( 80, 80, 80 );
+	qproperty-selectedColor: rgb( 0, 125, 255 );
 	qproperty-fgColor: rgb( 187, 227, 236 );
+	qproperty-fgMutedColor: rgb( 128, 128, 128 );
 	qproperty-textColor: rgb( 255, 255, 255 );
+	qproperty-textShadowColor: rgb( 0, 0, 0 );
+	qproperty-gradient: true;
 }
 
 /* sample track pattern */
 SampleTCOView {
 	color: rgb( 74, 253, 133 );
+	qproperty-mutedColor: rgb( 80, 80, 80 );
+	qproperty-selectedColor: rgb( 0, 125, 255 );
 	qproperty-fgColor: rgb( 187, 227, 236 );
+	qproperty-fgMutedColor: rgb( 128, 128, 128 );
```

Comment:
Is `:muted` a valid selector?  If so, that may certainly help.  I'm glad you're thinking about it thought because we do need to cleanup and standardize the `style.css` eventually, and you've quickly become the most active dev looking at that file. :+1: